pytransform3d/plot_utils/_plot_functions.py

```diff
@@ -94,33 +94,33 @@ def plot_box(ax=None, size=np.ones(3), A2B=np.eye(4), ax_s=1, wireframe=True,
 
 def plot_sphere(ax=None, radius=1.0, p=np.zeros(3), ax_s=1, wireframe=True,
                 n_steps=20, alpha=1.0, color="k"):
-    """Plot sphere.
+    """Plot sphere(s).
 
```

Comment:
Please describe the new feature here. It is a bit unusual, since no other function here supports plotting multiple objects.